src/tool.rs

```diff
@@ -105,6 +105,46 @@ impl Tool {
             .unwrap_or_default()
         }
 
+        fn guess_family_from_stdout(
+            stdout: &str,
+            path: &Path,
+            cargo_output: &CargoOutput,
+        ) -> Result<ToolFamily, Error> {
+            cargo_output.print_debug(&stdout);
+
+            // https://gitlab.kitware.com/cmake/cmake/-/blob/69a2eeb9dff5b60f2f1e5b425002a0fd45b7cadb/Modules/CMakeDetermineCompilerId.cmake#L267-271
+            // stdin is set to null to ensure that the help output is never paginated.
+            let accepts_cl_style_flags =
```

Comment:
Hmm, I'm wondering, can't we do this check _first_, and then use that to figure out whether to do `--` or not based on that?

Or, does `cl.exe` not support `--` either?

Reply:
Ah, yeah, it doesn't.